ota_pal.c

```diff
@@ -30,6 +30,19 @@
  *
  */
 
+#include <string.h>
+
+
+#include "logging_levels.h"
+/* define LOG_LEVEL here if you want to modify the logging level from the default */
+
+#define LOG_LEVEL    LOG_INFO
+
+#include "logging.h"
+
+
+#include "tls_transport_config.h"
```

Comment:
This header may not be available on some systems.

Reply:
Removed